### ISSUE TITLE
Dark UI theme

### ISSUE THREAD

**Issue by eman1986** [evolus/pencil#427]
* Operating system: All
* Pencil version: 3.0.4

not sure if this has been requested or if this is the right spot to post it, but I think a dark UI theme would be a great addition to this application.

**Comment by algiuxass** [comment]
It's been like 4 years, is there no dark theme yet?

**Comment by theapache64** [comment]
It's been like 4 years 10 months

**Comment by Mitch037** [comment]
I would like to bump up the request. It's been a while since the request was made. Maybe something can be done about introducing Dark Mode/Theme to Pencil Project?

**Comment by PhilipOfMacedon** [comment]
It seems this project is somewhat abandoned, considering the last major commits was 7 years ago and it is maintained by only 5 participants. But still someone will hopefully see this and help us out. Bump.

**Comment by theapache64** [comment]
> It's been like 4 years 10 months

It's been 6 years, and counting :(

**Comment by macarrp** [comment]
Here I was searching for Pencil dark mode and then I find this... Keep counting boys.. :(

**Comment by gbelmont22** [comment]
I hate sounding like 'that guy', but this definitely needs a dark theme.  Try staring into a blinding white screen with Fuchs' Dystrophy.

**Comment by bckelley** [comment]
7 years 10 months

**Comment by Lavshyak** [comment]
Bump. Dark theme would be great for people

**Comment by psi4j** [comment]
> Bump. Dark theme would be great for people

It's not gonna happen, bro. This is basically abandoned.

<img width="114" height="49" alt="Image" src="https://github.com/user-attachments/assets/618fae7b-24d4-4602-9bf9-cc4558ee40c9" />

**Comment by Lavshyak** [comment]
@psi4j do u know best analog?

**Comment by psi4j** [comment]
> @psi4j do u know best analog?

No, sorry. I just use my Remarkable tablet or a chalkboard for this sort of thing now. It's that or suffer your eyes with this thing. It is a good app, but light mode is brutal. Maybe try your luck with Mermaid diagrams and use them in a viewer.

**Comment by theapache64** [comment]
> > It's been like 4 years 10 months
> 
> It's been 6 years, and counting :(

7 years 10 months and i still miss dark mode

**Comment by Lavshyak** [comment]
I think  yworks /editors yED live is good enough
